packages/wpcom-checkout/src/checkout-labels.ts

```diff
@@ -93,5 +94,16 @@ export function getLabel( product: ResponseCartProduct ): string {
 	if ( product.meta && ( isDomainProduct( product ) || isDomainTransfer( product ) ) ) {
 		return product.meta;
 	}
+
+	if ( isJetpackAISlug( product.product_slug ) && product.quantity ) {
+		return translate( '%(productName)s (%(quantity)d requests per month)', {
+			args: {
+				productName: product.product_name,
+				quantity: product.quantity,
+			},
+			textOnly: true,
```

Comment:
Is there a reason that `textOnly` is on this one and not the one on `/me/purchases`?

Reply:
Yep, because the other one is expected to return a TranslateResult or string, but not this one 😂. So it's just me trying to avoid having to update a bunch of other functions